app/addons/auth/components/loginform.js

```
@@ -54,49 +55,76 @@ class LoginForm extends React.Component {
-              <Form.Control type="text"
-                id="username"
-                name="username"
-                ref={node => this.usernameField = node}
-                placeholder="Username"
-                onChange={this.onUsernameChange.bind(this)}
-                value={this.state.username} />
+        {this.state.showUserPasswordLogin && (
+          <form id="login" onSubmit={this.submit.bind(this)}>
+            <div className="row">
+              <div className="col12 col-md-5 col-xl-4 mb-3">
+                <label>Enter your username and password</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)